### ISSUE TITLE
AF: Clicking topic notification sends user to DB

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#1366]
User clicks this, gets taken to DB (including "take the DB tour" popping up). Should have been taken to the topic, or ideally, to the relevant idea. 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/eec3a208-75f6-4ee1-a564-186454667f77)

**Comment by BeccaMelhuish** [comment]
@ilmartyrk @ssin1901 moving this one to 'Next?' along with this similar one: https://github.com/citizenos/citizenos-fe/issues/1553

**Comment by BeccaMelhuish** [comment]
@ssin1901 I've got the feeling there may be several of this all very similar, relating to different actions relating to ideas (maybe to some others too). How would you feel about doing a bit of a QA review of the AF, and listing in one topic any occasions in which it is sending users to the wrong place (i.e. not to the topic in question). Then they could all be fixed at the same time. Not sure how much of a task that would be for you though, or could be automated? Will put it to QA dev for your consideration :)

**Comment by ssin1901** [comment]
@BeccaMelhuish it is doable but does it need to be done before the 19th or post 20th?

**Comment by BeccaMelhuish** [comment]
@ssin1901 Awesome thank you! It's a 'post launch' one so no big hurry, just when you have some space :)

**Comment by ssin1901** [comment]
@BeccaMelhuish this confused me, could I put it into the Parked column? I had to reread the issue to understand that it's just checking. Upvoting i.e "liking" the idea in Test took me to the topic as expected

**Comment by BeccaMelhuish** [comment]
Yes this one can go to parked indeed :)
